dev/DynamicDependency/MddLifetimeManagement.cpp

```diff
@@ -101,81 +101,90 @@ STDAPI MddLifetimeManagementGC() noexcept try
     winrt::Windows::Management::Deployment::PackageManager packageManager;
     winrt::hstring currentUser;
 
-    const UINT32 c_maxMajorVersion{ 9 };
+    const UINT16 c_majorMinorVersions[][3]{ // [n] = { Major, minMinor, maxMinor }
+        { 0, 8, 9 },
+        { 1, 0, 2 }
```

Comment:
how does this not become stale? Does this need to change over time? If so, can this be generated into a .h in a build script or similar? Will may have thoughts about how to tie this to other build metadata for valid version numbers.

Reply:
It's the DDLM cleanup support. Yes, it can go stale if we have 0.10 or 1.11 but it's also cross-version but we have no plans for 0.9+ and by the time we get to 1.11 or 2.1 I expect other changes in this space.

Making this work for current-version (1.0 implementation only cleans up 1.0.*, 1.1 only cleans up 1.1.*, etc) creates additional problems, e.g. we can't have 1 scheduled job doing the work, we need N (1 for each major.minor installed on the machine). Right now the API must be explicitly called. We don't have a recurring 'maintenance task' to tap this yet. @dhoehna and I've discussed that sort of maintenance-GC (he has similar needs for EnvironmentManager), plus there's the activity topic.

Changing this to only look for version.current poses additional complications for build (we lack some of that cross-pipeline coordination today) and testing (tests would need similar build-time not-constant support).

Given how the dyndep-build-time-overrides is shaping up a simple solution is starting to materialize. I'll make a note to review this for 1.0.